fix : extralabels warning on several pages

atm-greg opened the pull request:
# fix extralabel warning
The second parameter of the fetch_optionals method is optionnal and create warning in log files for several pages. No need to let the log file grow for this warning.

eldy committed:
Work on website import

eldy committed:
Add error message

eldy committed:
error message

eldy committed:
Merge pull request #9245 from grandoc/new_branch_14_08_2018

Standardize code

eldy committed:
Merge pull request #9247 from frederic34/missingReturn

return comment

eldy committed:
Merge branch 'develop' into trimtrailing

eldy committed:
Merge branch 'develop' into FunctionDeclaration

eldy committed:
Fix left join broken

eldy committed:
Fix escape

bugness-chl committed:
FIX adding GROUP BY for PostgreSQL

eldy committed:
Fix menu entry

eldy committed:
Fix menu entry

eldy committed:
Fix url in tooltip

eldy committed:
Fix url in tooltip

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

frederic34 committed:
Update list.php

eldy committed:
Use session cache for stats

eldy committed:
NEW Can add autorefresh=X in any URLs to refresh page after X seconds

eldy committed:
Merge branch '7.0' of git@github.com:Dolibarr/dolibarr.git into 8.0

eldy committed:
Merge branch '8.0' of git@github.com:Dolibarr/dolibarr.git into develop

Conflicts:
	htdocs/filefunc.inc.php

atm-greg committed:
fix : extralabels warning on several pages

eldy committed:
Work on import

eldy committed:
NEW Can export/import a website template

frederic34 committed:
Update actions_printing.inc.php

frederic34 committed:
Update actions_printing.inc.php

frederic34 committed:
Update propal.class.php

frederic34 committed:
Update fichinter.class.php

frederic34 committed:
Update fichinter.class.php

grandoc committed:
standardize code

grandoc committed:
Update and standardize code

grandoc committed:
Update and standardize code

grandoc committed:
Update and standardize code

grandoc committed:
Update and standardize code

eldy committed:
Fix missing translation

eldy committed:
Merge branch '8.0' of git@github.com:Dolibarr/dolibarr.git into develop

grandoc committed:
Typo

frederic34 committed:
Update propal.lib.php

grandoc committed:
Update html.formadvtargetemailing.class.php

torvista committed:
Main url auto-filled field was hardcoded as http: easy to overlook this when running installer via https.
This sets protocol to be the same as the installer page.

torvista committed:
Add description to database prefix field to avoid confusion when not specifying a database prefix.

eldy committed:
Fix trans

eldy committed:
Fix translation

eldy committed:
Module DAV

eldy committed:
Fix responsive

eldy committed:
FIX Look and feel v8

eldy committed:
FIX Delete file on smartphone

eldy committed:
Merge branch '8.0' of git@github.com:Dolibarr/dolibarr.git into develop

ptibogxiv committed:
fix duration display for service in list.php

eldy committed:
Fix date format

eldy committed:
Add control to avoid duplicate aliases

eldy committed:
Fix export/import web site

eldy committed:
Update list.php

eldy committed:
Update list.php

eldy committed:
Update list.php

eldy committed:
Merge pull request #9270 from ptibogxiv/patch-37

fix duration display for service in list.php

eldy committed:
Merge pull request #9260 from frederic34/patch-19

FIX propal.class.php

eldy commented:
The param become deprecated but code is not yet working everywhere without it in 7.0.
The goal is to have all call without param in develop so can you push this change in develop ?

eldy committed:
Merge pull request #9252 from bugness-chl/8.0-2

FIX adding GROUP BY for PostgreSQL

eldy committed:
Merge pull request #9264 from grandoc/new_branch_17_08_2018

standardize code

eldy committed:
Merge pull request #9259 from frederic34/patch-18

Update actions_printing.inc.php

eldy committed:
Merge pull request #9263 from frederic34/patch-20

Update fichinter.class.php

eldy committed:
Merge pull request #9266 from frederic34/patch-21

Update propal.lib.php

eldy committed:
Merge pull request #9253 from frederic34/patch-11

Update list.php

eldy committed:
Merge branch '8.0' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Merge pull request #9150 from frederic34/variant

code cleaning

eldy committed:
Update replenish.php

eldy committed:
Merge pull request #8975 from atm-greg/Add_hook_on_replenish

add hooks on stock replenish

eldy committed:
Update index.php

eldy committed:
Merge pull request #9058 from defrance/patch-123

New : Add total price on dashboard

eldy committed:
Merge branch 'develop' into trimtrailing

eldy committed:
Merge pull request #9248 from frederic34/FunctionDeclaration

function declaration

eldy committed:
Fix missing trans

eldy committed:
Fix cron task label

frederic34 committed:
Merge remote-tracking branch 'upstream/develop' into trimtrailing

eldy committed:
Trad

eldy committed:
Module builder is stable

eldy committed:
Prepare 9.0

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

frederic34 committed:
align parameters

eldy committed:
Fix urls in project module

eldy committed:
FIX badge on time spent on project and tasks

eldy committed:
Fix PHPCS

frederic34 committed:
Merge branch 'develop' into patch-18

eldy committed:
Fix PHPCS

eldy committed:
Clean comment

eldy committed:
Fix coding style

frederic34 committed:
Merge remote-tracking branch 'upstream/develop' into trimtrailing

frederic34 committed:
remove superfluous spaces

grandoc committed:
Update and standardize code

grandoc committed:
Update and standardize code

grandoc committed:
Update and standardize code

grandoc committed:
Update and standardize code

grandoc committed:
Update and standardize code

grandoc committed:
Update and standardize code

grandoc committed:
Update and standardize code

frederic34 committed:
Update antivir.class.php

frederic34 committed:
Update antivir.class.php

grandoc committed:
Update and standardize code

eldy committed:
FIX Get templates in a forced language

eldy committed:
FIX Language selection lost if error during creation of email template

eldy committed:
Fix tab

grandoc committed:
Update and standardize code

grandoc committed:
Update and standardize code

grandoc committed:
Update and standardize code

grandoc committed:
Update and standardize code

grandoc committed:
Update deplacementstats.class.php

grandoc committed:
Update accountancysystem.class.php

grandoc committed:
Update accountingaccount.class.php

grandoc committed:
Update adherentstats.class.php

grandoc committed:
Update bookmark.class.php

grandoc committed:
Update bookmark.class.php

grandoc committed:
Update actioncomm.class.php

grandoc committed:
Update cactioncomm.class.php

grandoc committed:
Update advtargetemailing.class.php

grandoc committed:
Update accountingaccount.class.php

frederic34 committed:
Update actions_ticket.class.php

frederic34 committed:
Update propal.lib.php

grandoc committed:
Update commonobject.class.php

grandoc committed:
Update fournisseur.facture.class.php

ptibogxiv committed:
fix sync user/member

add more fields to sync

ptibogxiv committed:
fix sync user/member

add more fields to sync

grandoc committed:
Update fournisseur.product.class.php

grandoc committed:
Update livraison.class.php

ptibogxiv committed:
Fix sync user->member

add more fields

ptibogxiv committed:
Fix sync user->member

add more fields

grandoc committed:
Merge branch 'develop' into new_branch_20_06_2018

grandoc committed:
Update and standardize code

grandoc committed:
Merge branch 'new_branch_21_08_2018' of github.com:grandoc/dolibarr into new_branch_21_08_2018

grandoc committed:
Standardize and update code

frederic34 committed:
Create index.html

grandoc committed:
Standardize and update code

grandoc committed:
Standardize and update code

frederic34 committed:
Update functions.lib.php

homer8173 committed:
There is a begin transaction without rollback 

In this Select, there is no need of this Begin Transaction on my understanding.
Don't know why this line code is here for.

eldy committed:
Fix hookmanager var

grandoc committed:
Standardize and update code

grandoc committed:
Standardize and update code

grandoc committed:
Standardize and update code

eldy committed:
Fix cleandata method of modulebuilder template of AAPI

eldy committed:
FIX remove internal property isextrafieldmanaged from API returns

eldy committed:
FIX api template for list in module builder

eldy committed:
Merge branch '7.0' of git@github.com:Dolibarr/dolibarr.git into 8.0

Conflicts:
	htdocs/core/class/html.formmail.class.php
	htdocs/projet/class/task.class.php
	htdocs/projet/tasks/task.php

eldy committed:
Merge branch '8.0' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
FIX API template to delete a record

eldy committed:
Fix back to page when canceling an edit in the module template

eldy committed:
FIX warning for late template invoices to remove when suspended

eldy committed:
Merge branch '8.0' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Fix lib in modulebuilder

eldy committed:
Fix bad link

eldy committed:
Fix bad link

eldy committed:
Fix duplicate load

eldy committed:
Remove colspan

eldy committed:
Fix syntax error

eldy committed:
Add tdclass to form confirm for more control

eldy committed:
Fix tdclass

eldy committed:
Fix tdclass

eldy committed:
Fix tdclass

eldy committed:
Add forcecombo on select language

eldy committed:
Fix preselected language

jtraulle committed:
Merge pull request #1 from Dolibarr/develop

Merge from upstream

frederic34 committed:
Update productlot_list.php

https://www.dolibarr.fr/forum/527-bugs-sur-la-version-stable-courante/62492-dolibarr-8-erreur-sql-avec-le-menu-lots-series

jtraulle committed:
New REST API Explorer branding and improvements

- Header of REST API Explorer has been Dolibarr colored
- API Methods are now sorted alphabetically
- API endpoints with same names are now sorted in this order (GET, POST, PUT, PATCH, DELETE)

jtraulle committed:
New REST API improvements to thirdparties bank accounts and gateways

- API endpoints has been added and documented to allow user to add, update and delete payment gateways like the one used by Stripe module (stored in table llx_societe_account)
- API endpoints for CompanyBankAccounts has been slightly modified to be harmonized with other API endpoints (this should not be a problem since these methods are not yet in Dolibarr latest stable version (aka 8.0))
- Some documentation has been improved (to be shown in REST explorer)
- societeaccount.class.php has been slightly modified to include missing $login member ; notnull value of member fk_website is now -1 to put it to NULL when updating the record.

jtraulle committed:
Opening brakets should be on new lines

jtraulle committed:
Keep params order in sync between @parm, @url annotations and function signature.

jtraulle committed:
Fix Travis CI Code reported coding style issues

jtraulle committed:
Fix of an oversight when swapping parameters

eldy committed:
Fix error message

ATM-Marc committed:
FIX: customer/supplier card: add simple tooltip to amount boxes

eldy committed:
Fix trans

eldy committed:
Fix trans

eldy committed:
Fix trans

torvista committed:
Merge pull request #1 from Dolibarr/develop

update fork

eldy committed:
Merge pull request #9287 from ptibogxiv/patch-38

FIX sync member->user

eldy committed:
Merge pull request #9288 from ptibogxiv/patch-39

FIX sync user->member

eldy committed:
Merge pull request #9279 from frederic34/patch-11

FIX Error message when sending a file with .virus extension

eldy committed:
Update productlot_list.php

eldy committed:
Merge pull request #9301 from frederic34/patch-26

FIX sql error

eldy committed:
Merge pull request #9294 from homer8173/patch-3

FIX There is a begin transaction without rollback

eldy committed:
Merge branch '7.0' of git@github.com:Dolibarr/dolibarr.git into 8.0

eldy committed:
Fix alter forgot

eldy committed:
Merge pull request #9292 from frederic34/patch-24

Create index.html

eldy committed:
Update actions_ticket.class.php

eldy committed:
Merge pull request #9285 from frederic34/patch-21

Update actions_ticket.class.php

eldy committed:
Merge pull request #9286 from frederic34/patch-22

Update propal.lib.php

eldy committed:
Merge pull request #9293 from frederic34/patch-25

NEW Add __DAY_TEXT__ and __MONTH_TEXT__ substitutions vars

eldy committed:
Merge branch '8.0' of git@github.com:Dolibarr/dolibarr.git into 8.0

eldy committed:
Merge pull request #9275 from frederic34/patch-18

align parameters

eldy committed:
Update api_thirdparties.class.php

eldy committed:
Merge pull request #9303 from jtraulle/develop

NEW REST API improvements

eldy committed:
Merge pull request #9278 from grandoc/new_branch_21_08_2018

Update and standardize code

eldy committed:
Merge branch '7.0' of git@github.com:Dolibarr/dolibarr.git into 8.0

Conflicts:
	htdocs/core/class/commondocgenerator.class.php

eldy committed:
Merge pull request #9265 from grandoc/new_branch_18_08_2018

Typo

eldy committed:
Merge pull request #9168 from frederic34/trimtrailing

Trimtrailing

eldy committed:
Merge branch '8.0' of git@github.com:Dolibarr/dolibarr.git into develop

Conflicts:
	htdocs/modulebuilder/index.php

eldy committed:
Merge pull request #8996 from grandoc/new_branch_20_06_2018

clean and update code

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into develop

eldy committed:
Merge

eldy committed:
Merge pull request #9307 from ATM-Marc/FIX_6.0_thirdparty_info

NEW On customer/supplier card, add simple tooltip to amount boxes

eldy committed:
Fix logo in public survey page

eldy committed:
Merge branch '7.0' of git@github.com:Dolibarr/dolibarr.git into 8.0

eldy committed:
Fix logo on public page of survey

torvista committed:
Merge pull request #2 from Dolibarr/develop

update 28/8/2018

frederic34 committed:
trimtrailing

frederic34 committed:
fix syntax error

frederic34 committed:
Build Badge for 8.0

eldy committed:
Fix missing language

eldy committed:
Fix regression in media editor in websites

eldy committed:
Fix translation

eldy committed:
Fix trans

eldy committed:
Fix type of invoice encoded twice

eldy committed:
Fix translation

eldy committed:
NEW Can set the datestart and dateend of cron job into module descriptor

eldy committed:
Fix regression

ptibogxiv committed:
Fix multicompany in ticket/list.php

ptibogxiv committed:
Fix multicompany in ticket/list.php

Conflicts:
	htdocs/ticket/class/ticket.class.php

frederic34 committed:
Merge remote-tracking branch 'upstream/develop' into trimtrailing2

atm-quentin committed:
NEW hook on dispatch order fourn

eldy committed:
Fix for compatibility with lines with no prices (some external modules)

eldy committed:
Fix regression in php style

frederic34 committed:
Merge remote-tracking branch 'upstream/develop' into trimtrailing2

eldy committed:
Fix date_start and date_end not saved on supplier invoice

Conflicts:
	htdocs/core/class/commonobject.class.php
	htdocs/fourn/class/fournisseur.facture.class.php
	htdocs/fourn/facture/card.php

eldy committed:
Fix date_start and date_end not saved on supplier invoice

eldy committed:
Fix phpcs

ptibogxiv committed:
fix deletion of stripe customer

ptibogxiv committed:
fix deletion of stripe customer

eldy committed:
Comment

eldy committed:
Fix phpcs

eldy committed:
Merge pull request #9315 from frederic34/patch-11

Build Badge for 8.0

eldy committed:
Merge pull request #9324 from ptibogxiv/patch-43

Fix multicompany in ticket/list.php

eldy committed:
Merge pull request #9309 from frederic34/trimtrailing2

last trimtrailing

eldy committed:
FIX #9328

eldy committed:
FIX #9328

eldy committed:
Merge branch '7.0' of git@github.com:Dolibarr/dolibarr.git into 8.0

Conflicts:
	README.md

eldy committed:
FIX #9258

eldy committed:
Merge branch '8.0' of git@github.com:Dolibarr/dolibarr.git into develop

Conflicts:
	.travis.yml
	htdocs/fourn/class/fournisseur.facture.class.php
	htdocs/product/fournisseurs.php
	htdocs/ticket/class/ticket.class.php

eldy committed:
Merge branch 'develop' of git@github.com:Dolibarr/dolibarr.git into
develop

Conflicts:
	htdocs/ticket/class/ticket.class.php

eldy committed:
Update dispatch.php

eldy committed:
Fix to avoid fetch when no accounting account defined

eldy committed:
Merge branch '8.0' into 7.0_new_dispatch

eldy committed:
Update dispatch.php

eldy committed:
Merge pull request #9326 from atm-quentin/7.0_new_dispatch

FIX hook on dispatch order fourn

eldy committed:
Merge pull request #9329 from ptibogxiv/patch-44

FIX deletion of stripe customer

eldy committed:
Merge branch '8.0' of git@github.com:Dolibarr/dolibarr.git into 8.0

eldy committed:
ident

eldy committed:
Update fileconf.php

eldy committed:
Merge pull request #9267 from torvista/develop

Installer: set protocol of autofill of main url to match installer protocol

eldy committed:
Fix phpcs

atm-greg committed:
Merge branch '8.0' of github.com:Dolibarr/dolibarr into fix_extralabels_warning

atm-greg committed:
Merge branch 'develop' of github.com:Dolibarr/dolibarr into fix_extralabels_warning
